src/app/lib/config/services/persian.js

```diff
@@ -52,49 +52,50 @@ const persian = {
     },
     consentBanner: {
       privacy: {
-        title: "We've updated our Privacy and Cookies Policy",
+        title: 'ما سیاست‌های حفظ حریم خصوصی و کوکی‌های خود را به روز کرده‌ایم',
         description: {
           uk: {
             first:
-              "We've made some important changes to our Privacy and Cookies Policy and we want you to know what this means for you and your data.",
+              'ما تغییرات مهمی در سیاست‌های حفظ حریم خصوصی و کوکی‌هایمان ایجاد کرده‌ایم و می‌خواهیم شما بدانید این تغییرات برای شما و اطلاعات مربوط به شما به چه معنی است',
             linkText: null,
             last: null,
             linkUrl: null,
           },
           international: {
             first:
-              "We've made some important changes to our Privacy and Cookies Policy and we want you to know what this means for you and your data.",
+              'ما تغییرات مهمی در سیاست‌های حفظ حریم خصوصی و کوکی‌هایمان ایجاد کرده‌ایم و می‌خواهیم شما بدانید این تغییرات برای شما و اطلاعات مربوط به شما به چه معنی است',
             linkText: null,
             last: null,
             linkUrl: null,
           },
         },
-        accept: 'OK',
-        reject: "Find out what's changed",
+        accept: 'تایید',
+        reject: 'ببنید چه تغییراتی ایجاد شده است',
         rejectUrl: 'https://www.bbc.co.uk/usingthebbc/your-data-matters',
       },
       cookie: {
-        title: 'Let us know you agree to cookies',
+        title: 'آیا با دریافت کوکی‌های ما موافقید؟',
         description: {
           uk: {
-            first: 'We use ',
-            linkText: 'cookies',
+            first: ' ما برای بهبود استفاده شما از خدمات بی‌بی‌سی از',
+            linkText: 'کوکی',
             last:
-              ' to give you the best online experience. Please let us know if you agree to all of these cookies.',
+              'استفاده می‌کنیم. اگر موافق هستید دریافت کوکی‌ها را تایید کنید ',
             linkUrl:
               'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
           },
           international: {
-            first: 'We and our partners use technologies, such as ',
-            linkText: 'cookies',
+            first:
+              ' ما و شریکانمان برای اینکه خدمات آنلاین بهتری به شما ارائه کنیم و محتوای مورد نظرتان را به شما نشان دهیم از تکنولوژی‌هایی مانند',
+            linkText: 'کوکی‌ها',
```

Comment:
Likewise here - without the spaces, this word 'cookies' will join up to the word following it. 
```suggestion
            linkText: ' کوکی‌ها ',
```
<img width="80" alt="Screen Shot cookie banner international" src="https://user-images.githubusercontent.com/3028997/59248679-01941b80-8c1b-11e9-963d-fc4014082fed.png">

Reply:
As above, moved the spaces to the opposite ends of 'first' and 'last'